uptane-standard.md

```diff
@@ -868,32 +873,37 @@ In order to perform partial verification, an ECU SHALL perform the following ste
 
 Full verification of metadata means that the ECU checks that the Targets metadata about images from the Director repository matches the Targets metadata about the same images from the Image repository. This provides resilience to a key compromise in the system.
 
-Full verification MAY be performed by either primary or secondary ECUs. The procedure is the same, except that secondary ECUs receive their metadata from the primary instead of downloading it directly. In the following instructions, whenever an ECU is directed to download metadata, it applies only to primary ECUs.
+Full verification MAY be performed by either primary or secondary ECUs.  Note that secondary ECUs receive their metadata from the primary instead of downloading it directly. In the following instructions, whenever an ECU is directed to download metadata, primary ECUs are to download it from the repositories, and secondary ECUs are to either request it from the primary ECU or refer to the cache of metadata already provided by the primary ECU.
```

Comment:
```suggestion
Full verification MAY be performed by either primary or secondary ECUs.  Note that secondary ECUs receive their metadata from the primary instead of downloading it directly. In the following instructions, whenever an ECU is directed to download metadata, primary ECUs are to download it from the repositories, and secondary ECUs are to either request it from the primary ECU, or refer to its cache of metadata previously downloaded from the primary ECU.
```